samples/tab-add-in-combined/nodejs/add-in/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "office-addin-taskpane",
+  "version": "0.0.1",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/OfficeDev/Office-Addin-TaskPane.git"
```

Comment:
Is this correct?

Reply:
Will cover in different new PR

Reply:
https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/965